test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -52,7 +52,6 @@ const (
 	windowsCommandTimeout                  = 1 * time.Minute
 	validateNetworkPolicyTimeout           = 3 * time.Minute
 	validateDNSTimeout                     = 2 * time.Minute
-	sshRetries                             = 3
```

Comment:
Strictly speaking we're not using this as a general const for "ssh retry" defaults (that is in the remote/ssh.go file)